esp/templates/survey/answers.html

```diff
@@ -48,9 +48,34 @@
             {% else %} 
                 There are no responses to this question.
             {% endif %}
+        </td> 
+
+        {% if q.question.get_params.list.count > 0 %} 
+        <!-- An if statement to check if the question type is labeled numeric rating. 
+        If this list returns a count greater than zero then that means this list contains labels. In this case,
+        it is assumed to be of labeled numeric rating type and not just numeric rating. -->
+        <td width = "100%">
+            <b>Statistics:</b>
+            <ul>
+                <li>Average: {{ q.answers|unpack_answers|average }}</li>
+                <li>Std. deviation: {{ q.answers|unpack_answers|stdev }}</li>
+                {% if q.question.per_class %}
+                    <li>Average for all classes: {{ q.question.global_average }}</li>
+                {% endif %}
+            </ul>
```

Comment:
It would be ideal if we could avoid this code duplication (should be able to just move the response labels portion to down below).

Reply:
Gotcha, I was confused and thought that numeric rating didn't have labels but I see that this is the question type with lower, middle, and upper labels.